### PR TITLE
added ALU84 from mechkeys.ca

### PR DESCRIPTION
**Pull request by TurboMech**
Created a stock ALU84 map for MACOS.

### CODE REVIEW

**Comment by jackhumbert** [comment]
Awesome! Would you mind updating the readme.md to follow [this template](https://docs.qmk.fm/documentation_templates.html#keyboard-readmemd-template)?

**Comment by TurboMech** [comment]
I sure will. Give me a day or so.

File: keyboards/alu84/kb.c [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "kb.h"
```
Comment (jackhumbert) [inline keyboards/alu84/kb.c]: These `kb` files should be named the same as the project - in this case, `alu84` :)

**Comment by jackhumbert** [comment]
It looks like you need to update your lib folder - `make git-submodule` should do this!

File: keyboards/alu84/kb.c (hunk 1 above)
Comment (TurboMech) [reply keyboards/alu84/kb.c]: will fix.

**Comment by TurboMech** [comment]
@jackhumbert hey I'm gonna update the mechkeys.ca mechmini 2 as well so everything will be corrected soon.

**Comment by jackhumbert** [comment]
Cool! Feel free to just reopen this PR when you're ready :)

It might work if you delete chibios folder, then run that!

**Comment by TurboMech** [comment]
@jackhumbert I figured it out had to update the homebrew stuff.  I'll get this stuff fixed this week and reopen the pull request.

**Pull request closed without merging**